pyroSAR/drivers.py

```diff
@@ -2398,51 +2441,52 @@ def close(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
     
-    def drop_element(self, scene, table):
+    def drop_element(self, scene):
         """
-        Drop an entry from the database.
+        Drop a scene from the data table.
 
         Parameters
         ----------
         scene: ID
             a SAR scene
-        table: str
-            either data or duplicates
 
         Returns
         -------
         """
-        if table == 'data':
-            # using schema to create SQL syntax
-            delete_statement = self.data_schema.delete().where(self.data_schema.c.scene == scene)
-        elif table == 'duplicates':
-            delete_statement = self.duplicates_schema.delete().where(self.duplicates_schema.c.scene == scene)
-        else:
-            raise ValueError("Only entries from table 'data' or 'duplicates' can be dropped")
-        if delete_statement:
-            # core SQL execution
-            self.conn.execute(delete_statement)
-            print('Entry with id {} was dropped!'.format(scene))
+        delete_statement = self.data_schema.delete().where(self.data_schema.c.scene == scene)
```

Comment:
Please also delete the scene from the `duplicates` table.

Reply:
should be working now